pb/state/state.proto

```diff
@@ -88,6 +88,9 @@ message AlertInfo {
   // Dynamic email list, route email alerts to these instead of the configured
   // defaults.
   repeated string email_addresses = 15;
+
+  // Maps (custom column headers name):(custom column headers value) for arbitrary alert properties.
+  map<string, string> custom_column_headers = 16;
```

Comment:
(Note: this applies to summary.proto too).

Since this is an alert, we're interested in column headers from two columns; the first column to fail, and the last column to pass. I think you either want to clarify which column this is pulled from (probably the first column to fail), or add fields for both (similar to `fail_build_id` and `pass_build_id` above). I'd recommend the former.